<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AssignmentReportWindow</class>
 <widget class="QMainWindow" name="AssignmentReportWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Отчет по назначениям</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnBack">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Тип фильтра:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbFilterType"/>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="lblPersonnel">
        <property name="text">
         <string>Сотрудник:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbPersonnel"/>
      </item>
      <item>
       <widget class="QLabel" name="lblDepartment">
        <property name="text">
         <string>Отдел:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbDepartment"/>
      </item>
      <item>
       <widget class="QLabel" name="lblPosition">
        <property name="text">
         <string>Должность:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbPosition"/>
      </item>
      <item>
       <widget class="QLabel" name="lblDateRange">
        <property name="text">
         <string>Период:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="deStartDate"/>
      </item>
      <item>
       <widget class="QLabel" name="lblTo">
        <property name="text">
         <string>до</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="deEndDate"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnSearch">
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget"/>
    </item>
    <item>
     <widget class="QLabel" name="lblStats">
      <property name="text">
       <string>Статистика</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar"/>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
